rust-windowing/winit
# Clean up various things for Keyboard v2

- [ ] Updated [feature matrix](https://github.com/rust-windowing/winit/blob/master/FEATURES.md), if new features were added or implemented

Various cleanups for Keyboard V2 (#2789):

- Use the build constants for the platforms.
- Make sure `memmap2` is at the latest version and is actually imported.
- Run rustfmt.
- Add documentation for `KeyCodeExtScancode`.
